libs/blocks/table/table.js

```diff
@@ -78,6 +78,27 @@ function handleHeading(table, headingCols) {
       col.innerHTML = '';
       col.append(row1, row2);
     }
+    const trackingHeader = col.querySelector('.tracking-header');
+    const nodeToApplyRoleScope = trackingHeader ?? col;
+
+    if (trackingHeader) {
+      const trackingHeaderID = `t${tableIndex + 1}-c${i + 1}-header`;
+      trackingHeader.setAttribute('id', trackingHeaderID);
+
+      const headerBody = col.querySelector('.body:not(.action-area)');
+      if (headerBody) headerBody.setAttribute('id', `${trackingHeaderID}-body`);
+
+      const headerPricing = col.querySelector('.pricing');
+      if (headerPricing) headerPricing.setAttribute('id', `${trackingHeaderID}-pricing`);
```

Comment:
Could we shorten this by using optional chaining here: `headerPricing?.setAttribute()`?

Reply:
Applied @drashti1712